apitools/base/py/credentials_lib.py

```diff
@@ -13,7 +14,9 @@
 import oauth2client.locked_file
 import oauth2client.multistore_file
 import oauth2client.service_account
-import oauth2client.tools  # for flag declarations
+# The `tools` import is rarely needed, but takes a long time.
```

Comment:
yeah, i vaguely remember running into this and not digging much. you can probably drop this whole comment now (since we use `tools` for the arg parser).